polygeist/tools/cgeist/Test/Verification/sycl/parallel_for.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: clang++ -fsycl -fsycl-targets=spir64-unknown-unknown-syclmlir %s -o %t.out 2>&1 | FileCheck %s --implicit-check-not="{{error|Error}}:"
+
+// RUN: clang++ -fsycl -fsycl-device-only -emit-llvm -fsycl-targets=spir64-unknown-unknown-syclmlir %s -o %t.bc 2>/dev/null
+
+// Test that the LLVMIR generated is verifiable.
+// RUN: opt -verify -disable-output < %t.bc
+
+// Verify that LLVMIR generated is translatable to SPIRV.
+// RUN: llvm-spirv %t.bc
+
+// RUN: llvm-dis %t.bc
+// RUN: cat %t.ll | FileCheck %s --check-prefix=LLVM
```

Comment:
Do we need the prefix here if we are only testing the LLVM output or are you planning on adding MLIR checks later, too?

Reply:
There are other `RUN` lines with `FileCheck`, so we cannot use `CHECK`.